pkg/cmd/cli/cmd/login/logout.go

```diff
@@ -123,25 +123,43 @@ func (o LogoutOptions) RunLogout() error {
 		return err
 	}
 
+	// attempt to always delete token from local config
+	// before deleting using api.
+	tokenDeleted, err := o.deleteTokenFromConfig()
+	if err != nil {
+		return err
```

Comment:
Is the ordering here important? Not sure we want to be in a situation where we have deleted the token from config but failed before trying to delete it using the API.